lynx/patterns/link-in-bio-dark-background.php

```diff
@@ -8,15 +8,15 @@
 ?>
 
 <!-- wp:cover {"customOverlayColor":"#011a0b","minHeight":100,"minHeightUnit":"vh","align":"full"} -->
-<div class="wp-block-cover alignfull" style="min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-100 has-background-dim" style="background-color:#011a0b"></span><div class="wp-block-cover__inner-container"><!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"50px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+<div class="wp-block-cover alignfull" style="min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-100 has-background-dim" style="background-color:#011a0b"></span><div class="wp-block-cover__inner-container"><!-- wp:group {"layout":{"inherit":true,"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:group {"layout":{"type":"flex","orientation":"vertical"},"style":{"spacing":{"blockGap":"50px"}}} -->
 <div class="wp-block-group"><!-- wp:site-logo {"className":"is-style-rounded"} /-->
 
-<!-- wp:group {"style":{"spacing":{"blockGap":"0px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+<!-- wp:group {"layout":{"type":"flex","orientation":"vertical"},"style":{"spacing":{"blockGap":"0px"}}} -->
 <div class="wp-block-group"><!-- wp:site-title {"isLink":false,"style":{"color":{"text":"#ffffffb3"},"typography":{"textTransform":"uppercase","fontStyle":"normal","fontWeight":"500","letterSpacing":"2px"}}} /-->
 
-<!-- wp:heading {"style":{"typography":{"textTransform":"uppercase","fontSize":"40px"},"color":{"text":"#dbff00"}},"className":"has-custom-font","fontFamily":"space-mono"} -->
-<h2 class="has-text-color has-space-mono-font-family has-custom-font" style="color:#dbff00;font-size:40px;text-transform:uppercase;"><?php echo  esc_html__( 'Musician &amp; Songwriter', 'lynx' ); ?></h2>
+<!-- wp:heading {"style":{"typography":{"textTransform":"uppercase"},"color":{"text":"#dbff00"}},"className":"has-custom-font","fontSize":"x-large","fontFamily":"space-mono"} -->
+<h2 class="has-custom-font has-text-color has-space-mono-font-family has-x-large-font-size" style="color:#dbff00;text-transform:uppercase"><?php echo  esc_html__( 'Musician & Songwriter', 'lynx' ); ?></h2>
```

Comment:
the class `has-custom-font` is no longer required.

Reply:
Thanks for the tip!